django/contrib/admin/sites.py

```diff
@@ -409,10 +409,10 @@ def login(self, request, extra_context=None):
         request.current_app = self.name
         return LoginView.as_view(**defaults)(request)
 
-    def _build_app_dict(self, request, label=None):
+    def get_app_dict(self, request, label=None):
```

Comment:
I don't see much value in renaming this method. This wouldn't make it a public API as explicit docs are also required. Also, there is a separate ticket-25671 and #13918 that propose to change related methods. I think we can revert this change for now.

Reply:
Yes, ticket #13918 and PR https://code.djangoproject.com/ticket/25671 is fine.